incubator/hnc/cloudbuild.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
```

Comment:
Is there a relatively easy way to add a step to pull the code directly from Github? Otherwise, I assume what happens is that you must check out the code locally on your workstation and invoke `make release` from there? Which is fine as a short-term solution but if it's easy to grab the source directly, that's even better.

If this is hard, can you add a TODO comment in both the makefile and the cloudbuild.yaml?

Reply:
There's a (relatively more common) way to use Cloud Build to build from Github repo for each commit/PR, but it doesn't seem common to manually trigger only one release at a certain point of time. I will add the TODO and leave it for a later discussion.